core/lib/generators/mno_enterprise/install/templates/config/newrelic.yml

```diff
@@ -0,0 +1,46 @@
+#
+# This file configures the New Relic Agent.  New Relic monitors Ruby, Java,
+# .NET, PHP, Python and Node applications with deep visibility and low
+# overhead.  For more information, visit www.newrelic.com.
+#
+# For full documentation of agent configuration options, please refer to
+# https://docs.newrelic.com/docs/agents/ruby-agent/installation-configuration/ruby-agent-configuration
+
+common: &default_settings
+  # Required license key associated with your New Relic account.
+  license_key: <%%= ENV['NEWRELIC_LICENSE_KEY'] %>
```

Comment:
This one is tricky :)

`<%%` will be rendered as a literal `<%`.

I want the resulting file to have `<%= ENV['NEWRELIC_LICENSE_KEY'] %>` and not to interpolate  to `ENV['NEWRELIC_LICENSE_KEY']` which would probably be `nil`

For the other `<%= Rails.application.class.parent_name %>`, I want it to be replaced by the app name when the file is generated.